src/css/profile/mobile/common/slider.less

```diff
@@ -6,18 +6,27 @@
 	flex-direction: column;
 	justify-content: space-around;
 
+	.ui-slider-bar::before {
+		content: '';
+		left: 0;
+		top: 0;
+		width: 100%;
+		height: 100%;
+		border-image: url(images/3_Controllers/tw_progress_bar_n.9.png) 3 fill;
+		border-image-width: @px_base;
```

Comment:
this entry seems like trick... can we write more clear: `1 * @px_base;`

Reply:
done